modules/web/src/app/cluster/list/cluster/component.ts

```diff
@@ -274,6 +274,7 @@ export class ClusterListComponent implements OnInit, OnChanges, OnDestroy {
 
   onActivate(component: QuotaWidgetComponent): void {
     component.showAsCard = false;
+    component.collapsible = true;
```

Comment:
May be not related where i have commented just wanted to show that we can add spacing 

extra space between **search bar** and **widget** thoughts?

<img width="665" alt="spacing" src="https://user-images.githubusercontent.com/17727069/210221848-b687825d-3b66-474d-bcc3-0e21f8d45ca0.png">

Reply:
**0**/2 **GB** 

Can you please recheck point-1 _":add "xxx/xxx unit" e. g. 10/10 GB tooltip hover over each project card/table quota"_

<img width="338" alt="units" src="https://user-images.githubusercontent.com/17727069/210222025-acb64297-0b5b-4a8c-a426-3a0518cdb947.png">